### PR TITLE
Views

### PR DESCRIPTION
**Pull request by Rayan84**
In this branch, the following content and features were added:

- View for users index, users show, posts index and posts show
- Logic to loop or filter the rendered data
- Style for the above mentioned view templates

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1,20 +1,40 @@
...
+        <span>Comments:</span>
+        <span><%= @comments.where(post_id: post.id).count %></span>
+        <span>Likes: </span>
+        <span><%= @likes.where(post_id: post.id).count %></span>
+      </p>
     </li>
   <% end %>
 </ul>
+  <div class="text-align-center">
+    <button class="see-all-posts-button">See all posts</button>
```
Comment (ahmetbozaci) [inline app/views/users/show.html.erb]: - [x] The button is not working. When ı click the see all posts button ı have to see the user's post as displayed on the wireframe. Please fix it.

**I can see users like in the wireframe, ı can click every user on the homepage, ı can see most recent posts after clicking the user. These are very good. But;**

- [x] You don't have to page that shows all posts after clicking the button. After creating this page you should also display `5 recent comments` as in the wireframe. 

image

- [x] Also, ı have to click every post on the page and see like wireframe
image

File: app/assets/stylesheets/users_show.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
+.page-container {
+  padding: 40px;
+}
+
+.see-all-posts-button {
+  padding-left: 18px;
+  padding-right: 18px;
+  padding-top: 8px;
+  padding-bottom: 8px;
```
Comment (ahmetbozaci) [inline app/assets/stylesheets/users_show.css]: ### OPTIONAL

- [x] For DRY CSS you can merge them in one padding.
- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and css best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️

### Hello @Rayan84  ✋🏼 
#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Nice to have 💯
- Your git-flow is correct ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️


___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] If you create example users, posts, comments in the `db/seeds.rb`  file that will be great. You can just migrate the seed and test it on the page.

___

### Cheers and Happy coding! 💻 👏

almost

___

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_


_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mentioning me as [ahmetbozaci](https://github.com/ahmetbozaci), I will receive a notification and will respond._
___
[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by Emmyn5600 — APPROVED** [review]
Hi @Rayan84 

STATUS: APPROVED :partying_face: 

Wooow👏, Your project is complete!  There is nothing else to say other than... it's time to merge it. 👏

Congratulations! 🎉 

Optional suggestions

-N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**